### PR TITLE
Add Comparative Aspects and Opinions Ranking for Recommendation Explanations (Companion) model

### PR DESCRIPTION
**Pull request by lthoang**
### Checklist:

- [ ] I have added tests.
- [x] I have updated the documentation accordingly.
- [x] I have updated `README.md` (if you are adding a new model).
- [x] I have updated `examples/README.md` (if you are adding a new example).
- [ ] I have updated `datasets/README.md` (if you are adding a new dataset).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,7 +143,8 @@ The table below lists the recommendation models/algorithms featured in Cornac. E
 
 | Year | Model and Paper | Type | Environment | Example |
 | :--: | --------------- | :--: | :---------: | :-----: |
-| 2024 | [Hypergraphs with Attention on Reviews (HypAR)](cornac/models/hypar), [docs](https://cornac.readthedocs.io/en/stable/api_ref/models.html#module-cornac.models.hypar.recom_hypar), [paper](https://doi.org/10.1007/978-3-031-56027-9_14)| Hybrid / Sentiment / Explainable | [requirements](cornac/models/hypar/requirements_cu116.txt), CPU / GPU | [quick-start](https://github.com/PreferredAI/HypAR)
+| 2024 | [Comparative Aspects and Opinions Ranking for Recommendation Explanations (Companion)](cornac/models/companion), [docs](https://cornac.readthedocs.io/en/stable/api_ref/models.html#module-cornac.models.companion.recom_companion) | Hybrid / Sentiment / Explainable | CPU | [quick-start](examples/companion_example.py)
```
Comment (tqtg) [inline README.md:146]: Do we have a paper reference for the model?

Comment (lthoang) [reply README.md]: The paper has been added for reference.

File: examples/README.md [hunk 2]
```diff
@@ -66,6 +66,8 @@
 
 [mter_example.py](mter_example.py) - Multi-Task Explainable Recommendation (MTER) with Amazon Toy and Games dataset.
 
+[companion_example.py](companion_example.py) - Comparative Aspects and Opinions Ranking for Recommendation Explanations (Companion) with Amazon Toy and Games dataset.
```
Comment (tqtg) [inline examples/README.md:69]: minor comment -- do we want to put it in the correct alphabetical order of the examples?

**Comment by tqtg** [comment]
@lthoang LGTM and congrats on the paper acceptance!

File: examples/README.md (hunk 2 above)
Comment (lthoang) [reply examples/README.md]: Updated.

**Review by tqtg — APPROVED** [review]

**Pull request merged**